### PR TITLE
trying to clone this so i can work on this 

### PR DESCRIPTION
**Pull request by huskerman34**
Dang i did something to where i cant  clone it

### CODE REVIEW

**Comment by huskerman34** [comment]
need to still work on fan page 5

**Comment by huskerman34** [comment]
Here is my case study and mod 5 had problems. I don't know why i keep merging these.

**Comment by huskerman34** [comment]
Mod 5 and case study 2 .

**Comment by huskerman34** [comment]
i think this should work

File: Neverwinter.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <li><strong>The Sword Coast </strong></li>
+            <p> Birdsong will travel to the swoardcoast to embark on his lifelong adventure. Travel to the Dale, fight the five headed dragon Tiamat,and travel to the Underdark to help your fellow drow the legendary Drizzit</p>
+        </ul>
+
+        <div id="content">
+            <strong>Contact Edgar F Greenwood</strong>
+            <a href="http://EdgarDrizzit.com"></a>
+        </div>
+      <link rel="stylesheet" href="neverwinterstyle.css">
+        <link rel="icon" href="favicon.ico" type="image/x-icon">
```
Comment (mbMosman) [inline Neverwinter.html]: These two link tags should be up in the head section near the top of the document.

File: Neverwinter.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <p>BirdSong is a Hunter Ranger from the Sun Elf Tribe.</p>
+            <li><strong>Trapper</strong></li>
+            <p> The trapper is support role player with more meele than long range fighter. </p>
+            <li><strong>The Sword Coast </strong></li>
+            <p> Birdsong will travel to the swoardcoast to embark on his lifelong adventure. Travel to the Dale, fight the five headed dragon Tiamat,and travel to the Underdark to help your fellow drow the legendary Drizzit</p>
+        </ul>
+
+        <div id="content">
+            <strong>Contact Edgar F Greenwood</strong>
+            <a href="http://EdgarDrizzit.com"></a>
```
Comment (mbMosman) [inline Neverwinter.html]: The contact link should be a mailto hyperlink

File: Neverwinter.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <title>Neverwinter Nights.</title>
+    <h1>Neverwinter </h1>
+</header>
+<div id="wrapper"></div>
+<nav>
+    Swordcoast &nbsp; Tiamat &nbsp; Icewind Dale &nbsp;
+</nav>
+
+
+<body id="body">
```
Comment (mbMosman) [inline Neverwinter.html]: The body tag needs to include all the visual content.  That includes the header and nav.  Everything that does not go into the `<head>` needs to be in the `<body>`.  This is a validation error.

File: neverwinterstyle.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+h1 {
+    height: 60px;
+    margin-top: 0;
+    color: #ff8000;
+    text-align: center;
+}
+footer {
+    font-size: 1.40em
+}
+title {
```
Comment (mbMosman) [inline neverwinterstyle.css]: The title is not a "visible" element.  It is part of the header.  So you cannot style it with CSS.  Use the header or h1 instead.

File: Neverwinter.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    <main>
+
+        <div id="dragon"></div>
+
+        <h2>Why Neverwinter</h2>
+
+        <p>Neverwinter is one of many stories from the Dungeon and Dragons series. I used to play the board game when i was little. I have read some of Drizzit stories from RA Salvatoire and Ed Greenwood. I like the Story line and the game play. Its more action based with less hot keys and the game is free to play. I think my favorite class is the Hunter Ranger. I am a trapper which is mainly support with some damage dealing and better survive ablility. Im up in your face Ranger. </p>
+        <img src="FrederickLorensiel_zpssebuojwx.jpg" alt="ranger" ; width="150" ; height="200" ;>
+        <h3>Birdsong Langer</h3>
```
Comment (mbMosman) [inline Neverwinter.html]: Unless this is a sub-heading of "Why Neverwinter", this should also be an h2 tag, not h3.

File: neverwinterstyle.css [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+}
+#wrapper {
+    width: 80%;
+    margin-left: auto;
+    margin-right: auto;
+}
+h2 {
+    text-shadow: 1px 1px 1px #ff0000;
+}
+h3 {color: #ffffff; color: rgba(255, 255, 255, 0.8); font-family: verdana, helvetica, sans-serif; font-size: 1.3em; padding-right: 10px; text-align: left;}
```
Comment (mbMosman) [inline neverwinterstyle.css]: The transparent color should be for the background, not the text.  It also asked for top/bottom padding on this heading along with rounded corners.

File: Neverwinter.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+            <li><strong>Hunter Ranger </strong></li>
+            <p>BirdSong is a Hunter Ranger from the Sun Elf Tribe.</p>
+            <li><strong>Trapper</strong></li>
+            <p> The trapper is support role player with more meele than long range fighter. </p>
+            <li><strong>The Sword Coast </strong></li>
+            <p> Birdsong will travel to the swoardcoast to embark on his lifelong adventure. Travel to the Dale, fight the five headed dragon Tiamat,and travel to the Underdark to help your fellow drow the legendary Drizzit</p>
+        </ul>
+
+        <div id="content">
+            <strong>Contact Edgar F Greenwood</strong>
```
Comment (mbMosman) [inline Neverwinter.html]: You should bold this using CSS, not the HTML `<strong>` tag.

**Pull request closed without merging**